apps/ai-image-tagging/lambda/tag.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { promisify } = require('util');
+
+const fetch = require('node-fetch');
+const AWS = require('aws-sdk');
+
+const IMAGES_BASE = 'https://images.ctfassets.net';
+
+const fetchImage = async imageUrl => {
```

Comment:
How is this affected by asset security?

Reply:
most likely won't work when we've got real asset secu and a customer is using it... but right now asset security = deleted assets delete public binary; no idea how/when real secu will be implemented